deployments/launcher/src/main/etc/indy/main.conf

```diff
@@ -84,6 +84,12 @@ storage.dir=${indy.home}/var/lib/indy/storage
 # mypool.threads=NN
 # mypool.priority=N
 
+; [folo]
```

Comment:
Need to remove the semi-colons.

Reply:
@sswguo those are commented out but for. some reason my IDE has put. semi colons in place of hashes.
BTW itshould be changed. to hashes on next. commit.